.github/workflows/build-vcpkg.yml

```diff
@@ -82,17 +82,10 @@ jobs:
           sudo apt-get install -y nodejs
 
       - name: OSX Dependencies
-        if: ${{ matrix.os == 'macos-10.15' }}
+        if: ${{ matrix.os == 'macos-11' }}
```

Comment:
This will require me to update the "required tests" setting again (note to self to remember to do so after merging this).

Reply:
Sorry about that - but I thought 10.15 was Big Sur until I double checked...